templates/projects/progress_reminder.html

```diff
@@ -0,0 +1,20 @@
+{% extends "projects/base.html" %}
+{% set active_page = "projects" %}
+{% set active_link  = "tasks" %}
+{% set section = _('Progress Reminder') %}
+{% from "_formhelpers.html" import render_field %}
+
+{% block projectcontent %}
+<h3>Send email notification on the progress of tasks</h3>
```

Comment:
*Fyi, this could be held off until we take on the Webhook URL notification task.*

As we may be adding adding notification options to this form, should we change this to a checkbox and rename it to "Enable email notification"?

This way, when we add additional options (such as the webhook), we would include subsequent checkboxes for each option. For example:

```
[X] Enable email notification
[X] Enable Webhook URL.
     [__________________________]

Notify when number of incomplete tasks drop to:
[_____ 14 _____]

[Set] | [Cancel]
```

Just brainstorming UI ideas. Thanks.

Reply:
Adding checkbox would require validations to apply sync between checkboxes and associated fields; disable field if checkbox unchecked, delete field values when checkbox value changed from checked to unchecked... 
As this page is dedicated to notifications, imo, current solution w/o checkbox would work - honor values when set, else ignore.

Reply:
That makes sense. However, when we do eventually add additional notifications options (webhook) we may need a way to disable the email notification option, as there is no specific email input field to differentiate when it should be active or disabled. :)

i.e., the text box for "Notify when the number of incomplete tasks ..." would apply to both email and webhook notifications.

Reply:
I'm fine whichever way feature owners would expect this. Will add my notes to jira and we can discuss with team.

Reply:
Or maybe rephrase the description as 
"notify when the number of incomplete tasks drops to the following number (leave empty to disable)"
same as webhook.

Reply:
Well, the idea is that once we have additional notification options, the user will want to be able to selectively toggle off/on specific notification methods. The user may only want the webhook, but not the email. However, for all methods, the textbox value would be required.

We can re-visit this redesign when we add more fields. :)

Until then, if we could just make the label more precise (i.e., "Notify when the number of remaining tasks is less than or equal to"). I'm not sure if it's necessary to mention leaving the value blank to disable (perhaps, that is implied?),